packages/api-bindings/src/lens/FollowModule.ts

```diff
@@ -17,6 +13,46 @@ export type ResolveFollowPolicy = {
   followModule: FollowModule | null;
 };
 
+export type ChargeFollowPolicy = {
+  type: FollowPolicyType.CHARGE;
+  amount: Erc20Amount;
+  recipient: string;
+  contractAddress: EvmAddress;
+  chainId: number;
+};
+
+export type NoFollowPolicy = {
+  type: FollowPolicyType.NO_ONE;
+  contractAddress: EvmAddress;
+  chainId: number;
+};
+
+/**
+ * @deprecated use {@link NoFollowPolicy} instead - removal slated for stable 2.x release
+ */
+export type NoFeeFollowPolicy = NoFollowPolicy;
+
+export type UnknownFollowPolicy = {
+  type: FollowPolicyType.UNKNOWN;
+  contractAddress: EvmAddress;
+  chainId: number;
+  initializeCalldata: Data;
+  initializeResultData?: Data;
+  signlessApproved: boolean;
+  sponsoredApproved: boolean;
+  verified: boolean;
+};
+
+export type OpenFollowPolicy = {
+  type: FollowPolicyType.ANYONE;
+};
+
+export type FollowPolicy =
+  | ChargeFollowPolicy
+  | NoFeeFollowPolicy
```

Comment:
it should be like this, no?
```suggestion
  | NoFollowPolicy
```